source/slang/slang-emit-spirv.cpp

```diff
@@ -4187,21 +4208,63 @@ struct SPIRVEmitContext
                 default:
                     break;
                 }
+                const auto opParent = parentForOpCode(opcode, parent);
```

Comment:
Not related to this IR, but the lambdas in this function is getting long enough that we should move them into standalone functions to make the code more readable.